### PR TITLE
Prose Polls june 13 2022

### PR DESCRIPTION
**Pull request by prose11**

### CODE REVIEW

File: governance/polls/Management of Velodrome Airdrop and veNFT - June 13, 2022.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+* Incentivize the DAI<>USDC and DAI<>OP pairs on Velodrome.
+* Send the resulting trading fees to the MakerDAO Optimism Governance Address.
+
+[For context](https://twitter.com/VelodromeFi/status/1529549901239050240), MakerDAO will be receiving an airdrop totaling ~1% of voting power to direct emissions on Velodrome.
+
+Please review the discussion [thread](https://forum.makerdao.com/t/signal-request-incoming-velodrome-airdrop-on-optimism-velo-venft/15371) to help inform your position before voting.
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
```
Comment (patrick-j-govalpha) [inline governance/polls/Management of Velodrome Airdrop and veNFT - June 13, 2022.md:37]: Do we have a Governance Address yet? If so, include it above in the Review section, if not, I'd include setting it up here as an outcome as we will need it set-up to enact the outcome of this poll

**Review by patrick-j-govalpha — CHANGES_REQUESTED** [review]
Just a few things to clarify but otherwise LGTM

File: governance/polls/Management of Velodrome Airdrop and veNFT - June 13, 2022.md (hunk 1 above)
Comment (prose11) [reply governance/polls/Management of Velodrome Airdrop and veNFT - June 13, 2022.md:37]: We do not at the moment :) adding

**Review by patrick-j-govalpha — APPROVED** [review]
LGTM!

**Pull request merged**